rustls/src/crypto/mod.rs

```diff
@@ -120,7 +120,7 @@ pub use crate::suites::CipherSuiteCommon;
 /// provider (dynamically).
 ///
 /// For example, if we want to make a provider that just overrides key loading in the config builder
-/// API ([`ConfigBuilder::with_single_cert`] etc.), it might look like this:
+/// API (with [`ConfigBuilder::with_single_cert`], etc.), it might look like this:
```

Comment:
Without these updates, the line seems to breaks right after `API (` in my browser.

With these updates, the line seems to break after `API (with` in my browser.

I think this is a slight improvement, wonder if anyone can think of something better for this.